.github/workflows/deploy.yml

```diff
@@ -21,11 +21,26 @@ jobs:
       - name: Install dependencies
         run: |
           sudo snap install --classic terraform
-          sudo snap install --classic juju-wait
+          # sudo snap install --classic juju-wait
       - name: Apply terraform
         run: |
           terraform init
           terraform apply -auto-approve -parallelism=1
           juju model-config -m openstack automatically-retry-hooks=true
-          juju-wait -vw -m openstack -r 10
+          juju wait-for application mysql --query='name=="mysql" && status=="active"' --timeout 10
```

Comment:
I think --timeout values all need a 's' - 10s for example.

The deploy check currently fails due to this.

Reply:
Done